modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -0,0 +1,135 @@
+
+. Navigate to the Mule upgrade tool directory.
+. Run the `upgrade` command:
++
+```
+$MULE_HOME/tools/upgrade-tool upgrade
+```
+////
+
+=== Upgrade Mule to a Specific Distribution
+
+To upgrade your Mule instance to a specific Mule distribution:
+
+. Navigate to the Mule upgrade tool directory.
+. Run the `upgrade -n` command, specifying the directory of the new Mule distribution as an argument:
++
+```
+$MULE_HOME/tools/upgrade-tool upgrade -n ./<mule-home-of-the-later-version>
+
+```
+
+== Backup and Rollback
+
+The Mule upgrade tool generates a backup of the current Mule instance before starting the upgrade process. The backup is stored in the `$MULE_HOME/.backup` folder. This backup allows the upgrade tool to restore a Mule instance to its previous state before performing the upgrade.
+
+=== Rollback an Upgrade
+
+To perform a rollback to the previous state before an upgrade:
+
+. Navigate to the Mule upgrade tool directory.
+. Run the `rollback` command:
+```
+$MULE_HOME/tools/upgrade-tool rollback
+```
+
+== Mule Upgrade Tool Commands and Options
+
+The following reference table shows a list of possible commands and options you can use when running the Mule upgrade tool:
+
+[%header%autowidth.spread,cols=".^a,.^a"]
+|===
+|Command | Description
+| `upgrade`
+| Upgrades the local Mule instance to the latest available version. +
+The following are the supported options for this command:
+
+[%header%autowidth.spread,cols=".^a,.^a"]
+!===
+  ! Options ! Description
+  ! `-n <newMuleDistributionPath>`
+  ! Upgrades the local Mule instance to a specific distribution that is downloaded and unpacked in your file system. Parameter `<newMuleDistribution>` specifies the path to the new Mule distribution.
+  +
+  For example:
+  +
+  `$ ./upgrade-tool upgrade -n /mule-home-of-the-later-version`
+  ! `-o <localMuleInstancePath>`
+  ! Upgrades the local Mule instance located in path `<localMuleInstancePath>`. This option enables you to upgrade a different Mule instance than the one located in the same `$MULE_HOME` path as the upgrade tool.
+  +
+  For example:
+  +
+  `$ ./upgrade-tool upgrade -o /mule-home-of-the-instace-to-upgrade`
+  ! `-d` ! Simulates the upgrade process without performing any permanent changes to the current Mule instance.
+  ! `-f` ! Performs the upgrade without requesting user confirmation for destructive operations like overwriting a backup or removing applied patches.
+!===
+| `rollback`
+| Restores the Mule instance to its previous state. This command works only after performing a successful upgrade with the tool. +
+The following are the supported options for this command:
+
+[%header%autowidth.spread,cols=".^a,.^a"]
+!===
+  ! Options ! Description
+  ! `-o <localMuleInstancePath>`
+  ! Restores the local Mule instance located in path `<localMuleInstancePath>`. This option enables you to restore a different Mule instance than the one located in the same `$MULE_HOME` path as the upgrade tool.
+  +
+  For example:
```

Comment:
ditto: GLOBALLY avoid putting "For example:" on a line by itself, as the sole introduction to an example or other nonnarrative element.
```suggestion
  ! Restores the local Mule instance located in path `<localMuleInstancePath>`. This option enables you to restore a different Mule instance than the one located in the same `$MULE_HOME` path as the upgrade tool, for example:
```